test/verify/check-machines-dbus

```diff
@@ -264,6 +264,18 @@ class TestMachinesDBus(machineslib.TestMachines):
         # Attach some interface
         m.execute("virsh attach-interface --persistent VmNotInstalled bridge virbr0")
 
+        # Change the os boot firmware configuration
+        if float(m.execute("virsh --version").strip()[:3]) >= 5.2:
```

Comment:
I don't like this much, as it could mean that we silently skip the test if that format ever changes. I think it would be better to explicitly have a list of OSes where EFI is *not* supported, so that over time we can reduce that list to zero.

Reply:
Replaced with the explicit list of OSes.

Reply:
OK. I'd rather have a list of non-working OSes, but it's not important. We can change it later when that blacklist becomes larger than the whitelist. Thanks for adding an `else:` so that we'll be notified when it starts to work on an image on refresh.